Lib/statistics.py

```diff
@@ -267,15 +267,15 @@ def _convert(value, T):
 
 
 def _find_lteq(a, x):
-    'Locate the leftmost value exactly equal to x'
+    """Locate the leftmost value exactly equal to x"""
```

Comment:
```suggestion
    'Locate the leftmost value exactly equal to x'
```
Comestics change are usually refused as they do not add value but cause useless code churn.

Reply:
I was a bit hesitant too about this cosmetic change. I will revert it back as you suggested in the next commit.

Let me leave here the reasons that I decided to make the change though:
* I have added unit tests specifically for those two functions, so it is not that I touch code in the codebase that it is not related with the proposed changes of this PR.
* They were the only functions in the `statistics` module that defined their docstrings with single quotes instead of triple double quotes.